lib/mcb/courses_editor.rb

```diff
@@ -59,12 +60,14 @@ def main_loop
     end
 
     def filter_single_course_options_if_necessary(choices)
-      choices.grep_v(->(c) { c.in?(["edit subjects"]) && @courses.count != 1 })
+      choices.grep_v(->(c) { c.in?(["edit subjects", "edit training locations"]) && @courses.count != 1 })
```

Comment:
Would it be worth using something more idiomatic like:

```suggestion
      choices.reject { |c| c.in?(["edit subjects", "edit training locations"]) && @courses.count != 1 }
```

The mechanism with `grep_v` seems to rely on `Proc#===` which apparently invokes `Proc#call` with the object being compared-to. Handy, but not as well understood as plain-old `reject`. :)

Reply:
thanks – done